changed the artifact name of the parent pom.xml to match the repo name

metas-ts opened the pull request:
i.e. changed it to metasfresh-dist.
goal: make sure that by default, the project is imported (into eclipse)
as "metasfresh-dist" that name is used in some .launch files, e.g. the
one to start the swing client. Thx @daveyx for bringing it up

metasfresh-dist parent pom.xml's artifact name shall be metasfresh-dist
https://github.com/metasfresh/metasfresh-dist/issues/7

metas-ts committed:
changed the artifact name of the parent pom.xml to match the repo name

i.e. changed it to metasfresh-dist.
goal: make sure that by default, the project is imported (into eclipse)
as "metasfresh-dist" that name is used in some .launch files, e.g. the
one to start the swing client. Thx @daveyx for bringing it up

metasfresh-dist parent pom.xml's artifact name shall be metasfresh-dist
https://github.com/metasfresh/metasfresh-dist/issues/7

daveyx commented:
now you was too fast for me :-) but i wanted to do exactly the same as you did...
only question now why jenkins not yet accept the new artifacts

metas-ts committed:
Merge remote-tracking branch 'origin/master' into gh7-metasfresh-dist

metas-ts committed:
build fix

metasfresh-dist parent pom.xml's artifact name shall be metasfresh-dist https://github.com/metasfresh/metasfresh-dist/issues/7

metas-ts commented:
Hi @daveyx 
> only question now why jenkins not yet accept the new artifacts

there was an error in the pom.xml i just fixed. let's see if other errors come up too

metas-ts committed:
fix groupId/nexus-URL in the jenkins file

metasfresh-dist parent pom.xml's artifact name shall be metasfresh-dist https://github.com/metasfresh/metasfresh-dist/issues/7
